libs/langgraph/src/constants.ts

```diff
@@ -58,7 +61,14 @@ export interface SendInterface {
 
 export function _isSendInterface(x: unknown): x is SendInterface {
   const operation = x as SendInterface;
-  return typeof operation.node === "string" && operation.args !== undefined;
+  return (
+    x != null &&
+    typeof x === "object" &&
+    "node" in x &&
+    "args" in x &&
```

Comment:
nit: just use `operation` consistently here?

Reply:
And actually, why was the previous guard not sufficient?

Reply:
> why was the previous guard not sufficient

This was the old one:

```typescript
export function _isSendInterface(x: unknown): x is SendInterface {
  const operation = x as SendInterface;
  return typeof operation.node === "string" && operation.args !== undefined;
}
```

`_prepareNextTask` was passing `undefined`, causing this guard to throw a `ReferenceError` on access of `operation.node`.

Side note: I'd have expected that the `typeof x === "object"` check to rule out null/undefined in later clauses of the statement. For some reason (likely `strictNullChecks` or similar in `compilerOptions`?) it doesn't, and the type checker gets angry on the `node in x` clause if I omit the `x != null`.

Reply:
will clean up the nit, though.